### PR TITLE
Eliminates androidx pre-release dependencies (no more ViewTreeLifecycleOwner)

### PR DESCRIPTION
**Pull request by rjrjr**
We can't be leaking RC dependencies as part of our public API in a world where
it's impossible to shade android libraries.

### CODE REVIEW

File: workflow-ui/radiography/src/androidTest/java/com/squareup/workflow1/ui/radiography/WorkflowViewRendererTest.kt [hunk 1]
```diff
@@ -1,102 +1,102 @@
...
-        container: ViewGroup?
-      ): View = View(contextForNewView).apply {
-        bindShowRendering(initialRendering, initialViewEnvironment) { _, _ ->
-          // Noop
-        }
-      }
-    }
-  }
-}
+// package com.squareup.workflow1.ui.radiography
```
Comment (rjrjr) [inline workflow-ui/radiography/src/androidTest/java/com/squareup/workflow1/ui/radiography/WorkflowViewRendererTest.kt:1]: @zach-klippenstein Would you rather I deleted this?

**Review by steve-the-edwards — APPROVED** [review]

File: workflow-ui/radiography/src/androidTest/java/com/squareup/workflow1/ui/radiography/WorkflowViewRendererTest.kt (hunk 1 above)
Comment (rjrjr) [reply workflow-ui/radiography/src/androidTest/java/com/squareup/workflow1/ui/radiography/WorkflowViewRendererTest.kt:1]: Answer is yes.

File: buildSrc/src/main/java/Dependencies.kt [hunk 2]
```diff
@@ -28,7 +28,6 @@ object Dependencies {
 
     object Lifecycle {
       const val ktx = "androidx.lifecycle:lifecycle-runtime-ktx:_"
-      const val viewModel = "androidx.lifecycle:lifecycle-viewmodel:_"
       const val viewModelKtx = "androidx.lifecycle:lifecycle-viewmodel-ktx:_"
```
[the remark this answers is not in the document]
Comment (rjrjr) [reply buildSrc/src/main/java/Dependencies.kt:31]: Actually, we were already using `viewModelKtx` long before this work started. I'm surprised that I was able to delete `viewModel`.

[the remark this answers is not in the document]
Comment (rjrjr) [reply buildSrc/src/main/java/Dependencies.kt:31]:
```
~/Development/square/workflow-kotlin ray/less-is-more + git grep Lifecycle.viewModel
samples/hello-workflow-fragment/build.gradle.kts:  implementation(Dependencies.AndroidX.Lifecycle.viewModelKtx)
samples/hello-workflow-fragment/build.gradle.kts:  implementation(Dependencies.AndroidX.Lifecycle.viewModelSavedState)
samples/hello-workflow/build.gradle.kts:  implementation(Dependencies.AndroidX.Lifecycle.viewModelKtx)
samples/hello-workflow/build.gradle.kts:  implementation(Dependencies.AndroidX.Lifecycle.viewModelSavedState)
```

**Comment by rjrjr** [comment]
So great: all espresso shards passed, but Assemble & Dokka timed out. :this-is-fine:

```
Caused by: java.lang.RuntimeException: Exception while loading package-list from ExternalDocumentationLinkImpl(url=https://docs.oracle.com/javase/6/docs/api/, packageListUrl=https://docs.oracle.com/javase/6/docs/api/package-list)
```

**Pull request merged**